language/tools/move-cli/src/utils/credential.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
Please rename this to `movey_credential.rs` or something else that indicates the connection to movey.